packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats-manual-modal.tsx

```diff
@@ -1,16 +1,30 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Icon, Loading, Modal, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { getUrlBase, isMobile } from '@deriv/shared';
 import 'Sass/app/modules/contract/accumulators-stats.scss';
+import { useStore } from '@deriv/stores';
 
-const AccumulatorsStatsManualModal = ({ icon_classname, is_dark_theme, is_manual_open, title, toggleManual }) => {
+type TAccumulatorsStatsManualModal = {
+    icon_classname: string;
+    is_dark_theme?: ReturnType<typeof useStore>['ui']['is_dark_mode_on'];
```

Comment:
@henry-deriv let's not rely on what useStore returns until we actually start using useStore in this file.🙂 please change to boolean